File: CS/E965.Module/BusinessObjects/FileSystemLinkObjectDemo.cs
using DevExpress.Xpo;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using FileSystemData.BusinessObjects;

namespace E965.Module.BusinessObjects {
    [DefaultClassOptions]
    [FileAttachment("File")]
    public class FileSystemLinkObjectDemo : BaseObject {
        public FileSystemLinkObjectDemo(Session session) : base(session) { }
        [Aggregated, ExpandObjectMembers(ExpandObjectMembers.Never), ImmediatePostData]
        public FileSystemLinkObject File {
            get { return GetPropertyValue<FileSystemLinkObject>("File"); }
            set { SetPropertyValue<FileSystemLinkObject>("File", value); }
        }
    }
}

File: VB/E965.Module/BusinessObjects/FileSystemLinkObjectDemo.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo
Imports FileSystemData.BusinessObjects

Namespace E965.Module.BusinessObjects
	<DefaultClassOptions, FileAttachment("File")>
	Public Class FileSystemLinkObjectDemo
		Inherits BaseObject

		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		<Aggregated, ExpandObjectMembers(ExpandObjectMembers.Never), ImmediatePostData>
		Public Property File() As FileSystemLinkObject
			Get
				Return GetPropertyValue(Of FileSystemLinkObject)("File")
			End Get
			Set(ByVal value As FileSystemLinkObject)
				SetPropertyValue(Of FileSystemLinkObject)("File", value)
			End Set
		End Property
	End Class
End Namespace
